app.py

```diff
@@ -0,0 +1,12 @@
+from flask import Flask
+
+app = Flask(__name__)
```

Comment:
is flask the best thing here? Try Django next

Reply:
Resolved.